src/drivers/mct/main/prep_glc_mod.F90

```diff
@@ -444,6 +446,31 @@ end subroutine prep_glc_map_one_field_lnd2glc
 
   !================================================================================================
 
+  subroutine prep_glc_zero_fields()
+
+    !---------------------------------------------------------------
+    ! Description
+    ! Set glc inputs to zero
```

Comment:
I realize this is just to generate consistent 'garbage' for times when the field is not valid.  One thought would be to fill it with a very obvious garbage value to help protect against the possibility that GLC accidentally tries to actually use it.  But I realize just filling with zeroes might be more aesthetically pleasing and such an approach should not be necessary, so feel free to ignore this comment.